lego/apps/comments/models.py

```diff
@@ -27,5 +27,14 @@ class Meta:
         ordering = ("created_at",)
         permission_handler = CommentPermissionHandler()
 
+    def delete(self):
+        with transaction.atomic():
```

Comment:
Is there any reason to open a transaction here?

Reply:
I guess not really.